templates/user/profile.html

```diff
@@ -79,6 +80,15 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
   })
   localizeField.addEventListener('change', event => event.target.checked && localize())
 
+  const publishField = document.querySelector('[name=publish]')
```

Comment:
because if I use by `name` that just take the firstbecause if I use by `name` that just take the first